mods/ra/maps/soviet-13a/soviet13a-AI.lua

```diff
@@ -0,0 +1,170 @@
+--[[
+   Copyright (c) The OpenRA Developers and Contributors
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+WTransWays =
+{
+	{ EastWaterEntry.Location, EastBeach1.Location },
+	{ EastWaterEntry.Location, EastBeach2.Location },
+	{ EastWaterEntry.Location, EastBeach3.Location },
+	{ WestWaterEntry.Location, WestBeach1.Location },
+	{ WestWaterEntry.Location, WestBeach2.Location }
+}
+
+WTransUnits =
+{
+	hard = { { "2tnk", "2tnk", "e3", "e3", "e3" }, { "2tnk", "2tnk", "2tnk", "2tnk" } },
+	normal = { { "2tnk", "1tnk", "e3", "e3", "jeep"  }, { "2tnk", "2tnk", "1tnk", "jeep" } },
+	easy = { { "2tnk", "e1", "e1", "e3", "e3" }, { "1tnk", "1tnk", "jeep", "jeep" } }
+}
+
+WTransDelays =
+{
+	easy = DateTime.Minutes(5),
+	normal = DateTime.Minutes(4),
+	hard = DateTime.Seconds(3)
```

Comment:
```suggestion
	hard = DateTime.Minutes(3)
```

This change definitely needs to be reverted 😅